pkgs/applications/misc/redis-desktop-manager/breakpad_ucontext.patch

```diff
@@ -0,0 +1,223 @@
+Submodule 3rdparty/gbreakpad contains untracked content
```

Comment:
Did you write this patch on your own?

Reply:
Backport of an upstream patch. Do I need to insert Copyright?

Reply:
No, this question was more related to https://github.com/NixOS/nixpkgs/pull/32100#issuecomment-347130991